www/blog/2019/05/13/geonames/index.md

```diff
@@ -0,0 +1,223 @@
+---
+layout: page
+title: New GeoNames-sourced locality records
+published: false
+date: 2019-05-13
+permalink: /blog/2019/05/13/geonames/
+category: blog
+excerpt: "We've recently added millions of records sourced from GeoNames, bringing global locality coverage to Who's On First."
+authors: [stepps00]
+image: "images/village.jpg"
+tag: [whosonfirst,wof,geonames,data]
+---
+
+We've spent a great deal of time importing and assessing data quality of new [Who's On First](https://whosonfirst.org/) locality records. More specifically, we've recently imported new locality records sourced from the [GeoNames](https://www.geonames.org/) database.
+
+Prior to this work, we've imported tens of thousands of new county records in Who's On First via [Mesoshapes](https://whosonfirst.org/blog/2016/12/08/mesoshapes/), added properties from [Statoids](https://whosonfirst.org/blog/2017/09/19/introducing-statoids/), and imported smaller amounts of locality records via the Quattroshapes point gazetteer, but we are happy to announce that Who's On First now has comprehensive global coverage at the locality placetype .
+
+Before this work began, the locality record count in Who's On First was just over 345,000. At the completion of this import, the locality record count increased more than ten-fold, to just over 4.4 million records. This locality import increased the total number of administrative records in the [whosonfirst-data repository](https://github.com/whosonfirst-data/whosonfirst-data) to 4.8 million. This detailed coverage at the locality level, in addition to the other placetype updates we've completed, give millions of detailed administrative hierarchies for cities, poplulated places, villages, and towns around the world.
```

Comment:
Please lead with this paragraph as the key take away. 

(Will require switch the paragraph order around, and update a few phrases and transitions.)

Reply:
Updated the first three paragraphs in recent commit.